mne/io/reference.py

```diff
@@ -97,11 +128,16 @@ def _apply_reference(inst, ref_from, ref_to=None):
                 'using the apply_proj() method function before the desired '
                 'reference can be set.'
             )
+
     for i in projs_to_remove:
         del inst.info['projs'][i]
```

Comment:
If possible: when calling `set_bipolar_ref` with an un-applied average reference projector in the `info['projs']` list, update the average reference projector to exclude the newly created bipolar channel (with a warning). If this is not possible: fail loud and don't allow it.